docs/earthfile/earthfile.md

```diff
@@ -149,7 +149,7 @@ To avoid any ambiguity regarding whether an argument is a `RUN` flag option or p
 
 #### Options
 
-##### `--push`
+``##### `--push`
```

Comment:
is this intentional?

![image](https://user-images.githubusercontent.com/1806823/126540278-bd1ce0f8-d17a-4ba8-8319-e3e4db00133d.png)

Reply:
I can't type with this brace on my finger :(